.circleci/config.yml

```diff
@@ -20,13 +20,11 @@ jobs:
   ruby-2.6.3-rails-5.2:
     docker:
     - image: circleci/ruby:2.6.3
-      environment:
-        BUNDLE_GEMFILE: Gemfile.rails-5.2
     working_directory: "~/stitches"
     steps:
     - checkout
     - run: bundle config stitchfix01.jfrog.io $ARTIFACTORY_USER:$ARTIFACTORY_TOKEN
-    - run: bundle install --full-index
+    - run: bundle install --full-index --gemfile=Gemfile.rails-5.2
```

Comment:
I changed the CircleCI config because specifying `BUNDLE_GEMFILE` will override it for all uses of `bundle
install` including when we're bundling the gems in the test application.